.storybook/preview.js

```diff
@@ -1,57 +1,63 @@
 // .storybook/preview.js
 
 import { breakpoints } from '@guardian/src-foundations';
-
+import { MINIMAL_VIEWPORTS } from '@storybook/addon-viewport';
 
 const customViewports = {
-    desktop: {
-      name: 'Desktop',
-      styles: {
-        width: breakpoints.desktop,
-        height: '963px',
-        },
-    },    
-    tablet: {
-      name: 'Tablet',
-      styles: {
-        width: breakpoints.tablet,
-        height: '963px',
-        },
-    },      
-    phablet: {
-      name: 'Phablet',
-      styles: {
-        width: breakpoints.phablet,
-        height: '963px',
-        },
-    },    
-    mobileLandscape: {
-      name: 'Mobile Medium',
-      styles: {
-        width:  breakpoints.mobileLandscape,
-        height: '963px',
-        },
-    },     mobileMedium: {
-      name: 'Mobile Medium',
-      styles: {
-        width:  breakpoints.mobileMedium,
-        height: '963px',
-        },
-    },    
-    mobile: {
-      name: "Mobile",
-      styles: {
-        width: breakpoints.mobile,
-        height: '963px',
-        },
-    },
+	desktop: {
+		name: 'Desktop',
+		styles: {
+			width: `${breakpoints.desktop}px`,
+			height: '100%',
+		},
+		type: 'desktop',
+	},
+	tablet: {
+		name: 'Tablet',
+		styles: {
+			width: `${breakpoints.tablet}px`,
+			height: MINIMAL_VIEWPORTS.tablet.styles.height,
+		},
+		type: 'tablet',
+	},
+	phablet: {
+		name: 'Phablet',
+		styles: {
+			width: `${breakpoints.phablet}px`,
+			height: MINIMAL_VIEWPORTS.mobile2.styles.height,
+		},
+		type: 'other',
+	},
+	mobileLandscape: {
+		name: 'Mobile Landscape',
+		styles: {
+			width: `${breakpoints.mobileLandscape}px`,
+			height: MINIMAL_VIEWPORTS.mobile2.styles.width,
+		},
+		type: 'mobile',
+	},
+	mobileMedium: {
+		name: 'Mobile Medium',
+		styles: {
+			width: `${breakpoints.mobileMedium}px`,
+			height: MINIMAL_VIEWPORTS.mobile2.styles.height,
+		},
+		type: 'mobile',
+	},
+	mobile: {
+		name: 'Mobile',
+		styles: {
+			width: `${breakpoints.mobile}px`,
+			height: MINIMAL_VIEWPORTS.mobile1.styles.height,
+		},
+		type: 'mobile',
+	},
 };
 
 export const parameters = {
-    layout: 'fullscreen',
-    viewport: {
-        viewports: {
-            ...customViewports
-        }
-    },
-};
\ No newline at end of file
+	layout: 'fullscreen',
```

Comment:
Don't think we need this - I've added it just to the article stories, everything else is fine without it.

Reply:
done 👍